sycl/test/vitis/simple_tests/loop_annotations.cpp

```diff
@@ -0,0 +1,37 @@
+// REQUIRES: vitis
+
+// RUN: rm -rf %t.dir && mkdir %t.dir && cd %t.dir
+// RUN: %clangxx -std=c++20 -fsycl -fsycl-targets=%sycl_triple %s -o %t.dir/exec.out
+// RUN: %ACC_RUN_PLACEHOLDER %t.dir/exec.out
+
+#include <sycl.hpp>
```

Comment:
Curious it does even compile. I think this bug was fixed upstream a few weeks ago.
`#include <sycl/sycl.hpp>`